res/css/views/rooms/_RoomBreadcrumbs2.scss

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@keyframes breadcrumb-popin {
+    0% {
+        // Ideally we'd use `width` instead of `opacity`, but we only
+        // have 16 nanoseconds to render the frame, and width is expensive.
+        opacity: 0;
+        transform: scale(0);
+    }
+    100% {
+        opacity: 1;
+        transform: scale(1);
+    }
+}
+
+.mx_RoomBreadcrumbs2 {
+    // Create a flexbox for the crumbs
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    width: 100%;
+
+    .mx_RoomBreadcrumbs2_crumb {
+        margin-right: 8px;
+        width: 32px;
+
+        // React loves to add elements, so only target the one we want to animate
+        &:first-child {
+            animation: breadcrumb-popin 0.3s;
```

Comment:
Does it make sense to get a design review of the animation now for this part...? I'm assuming you chose the animation style here? I worry it might get forgotten / lost if there's only one big design review at the end.

Reply:
Possibly, yea. I'll send a cleaner video to the team to review.

Reply:
the design team would like it to slide in from the left as a whole using `cubic-bezier(0.66,0.02,0.36,1)`. No opacity or scaling.